lib/json_schema/validator.rb

```diff
@@ -249,6 +249,12 @@ def validate_items(schema, data, errors, path)
           false
         else
           valid = true
+          if data.size > schema.items.count && schema.additional_items.is_a?(Schema)
+            (schema.items.count..data.count-1).each do |i|
+              valid = strict_and valid,
+                validate_data(schema.additional_items, data[i], errors, path + [i])
+            end
+					end
```

Comment:
Probably an editor gone haywire, but your `end` on this line has migrated out to the frontier. Would you mind tightening that up?